<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventInputWindow</class>
 <widget class="QMainWindow" name="EventInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAddEvent">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Додати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveEvent">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Видалити</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Назва події</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ймовірність</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSaveEvents">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>491</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
